Error running tests, reference to uniqueid.h

### Description
Testing components for example `storage` drop us:
```sh
In file included from ... m4a-firmware/RIOT/cpu/cortexm_common/vectors_cortexm.c:32:
... m4a-firmware/boards/m4a-mb/include/board.h:25:10: fatal error: uniqueid.h: No such file or directory
    25 | #include "uniqueid.h"
```
#### The error appears building the test suite for the following boards:
- **m4a-24g**
- **m4a-mb**
- 
#### Also testing the following components:
-  ds18
- storage
- serialization
- unittests

### Steps to reproduce the issue
- Go to `/tests/storage`
- Execute `make test flash`
- 
### Expected results
No errors

### Actual results
Errors

#### Versions

--- a/boards/m4a-24g/include/board.h
+++ b/boards/m4a-24g/include/board.h
@@ -25,7 +25,6 @@
 #include "periph_conf.h"
 #include "periph_cpu.h"
 #include "periph/cpuid.h"
-#include "uniqueid.h"
 
 #ifdef __cplusplus
 extern "C" {

--- a/boards/m4a-mb/include/board.h
+++ b/boards/m4a-mb/include/board.h
@@ -22,7 +22,6 @@
 #include "periph_conf.h"
 #include "periph_cpu.h"
 #include "periph/cpuid.h"
-#include "uniqueid.h"
 
 #ifdef __cplusplus
 extern "C"

--- a/boards/m4a-wroom/include/board.h
+++ b/boards/m4a-wroom/include/board.h
@@ -24,7 +24,6 @@
 #include "periph_conf.h"
 #include "periph_cpu.h"
 #include "periph/cpuid.h"
-#include "uniqueid.h"
 
 /**
  * @name    Button pin definitions

--- a/boards/m4a-wrover/include/board.h
+++ b/boards/m4a-wrover/include/board.h
@@ -22,7 +22,6 @@
 #include "periph_conf.h"
 #include "periph_cpu.h"
 #include "periph/cpuid.h"
-#include "uniqueid.h"
 
 /**
  * @name    LED (on-board) configuration

